lib/http_server_fixture/routes.ex

```diff
@@ -5,9 +5,11 @@ defmodule HTTPServerFixture.Routes do
   alias HTTPServerFixture.SimpleGetWithBody
   alias HTTPServerFixture.SimplePost
   alias HTTPServerFixture.SimpleHead
+  alias HTTPServer.Handlers.NotFound
 
   def route(%Request{path: "/echo_body"} = req), do: SimplePost.handle(req)
   def route(%Request{path: "/simple_get"} = req), do: SimpleGet.handle(req)
   def route(%Request{path: "/simple_get_with_body"} = req), do: SimpleGetWithBody.handle(req)
   def route(%Request{path: "/head_request"} = req), do: SimpleHead.handle(req)
+  def route(%Request{path: _path} = req), do: NotFound.handle(req)
```

Comment:
Just like we would not want to put the burden on the Handlers of handing 404 not founds, we would also not want to put the burden on the Routes file.  The server should be handling this, not the configuration.

Reply:
Keep pushing further (like into the `Router`